recipes/libssh2/all/test_package/CMakeLists.txt

```diff
@@ -1,10 +1,7 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
-
-find_package(Libssh2 REQUIRED CONFIG)
+find_package(libssh2 REQUIRED CONFIG)
```

Comment:
This change of the `find_package` name from `Libssh2` to `libssh2` is going to break some people. I have tried several ways of keeping `Libssh2` but for some reason my attempts have failed in CI (works locally for me). If someone knows how to fix this, please do so. Better to not break compatibility, even through I think the lowercase version is more consistent and less surprising.